.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,21 @@
+# Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
+blank_issues_enabled: false
+contact_links:
+- name: Create an **announcement**
```

How about making it tagged with smth like this?
```suggestion
- name: >-
    [ONLY FOR ADMINS]
    📢 Create an **announcement**
```
Also, maybe this item should be at the bottom of the list?

---

That would be wrong - anyone should be able to make announcements, not just admins or a selected group of folks.

Moving it to the bottom is probably a good idea though, to make it less likely to be abused.